src/Sulu/Bundle/AdminBundle/Resources/js/views/Tabs/Tabs.js

```diff
@@ -134,8 +143,27 @@ class Tabs<T> extends React.Component<Props<T>> {
                     <TabsComponent onSelect={this.handleSelect} selectedIndex={selectedTabIndex}>
                         {this.sortedTabRoutes.map((tabRoute) => {
                             const tabTitle = tabRoute.options.tabTitle;
+                            const tabBadges = tabRoute.options.tabBadges;
```

Comment:
using `[]` would simplify the logic below

Reply:
I don't know what you are meaning

Reply:
i forgot some word above 😓  
using `const tabBadges = tabRoute.options.tabBadges || []` would allow to omit the if condition below